Flandstation - Service Pack 4 (More QoL changes)

PestoVerde322 opened the pull request:
## About The Pull Request

Here it's the fourth Service pack for Flandstation, this version will feature changes in more neglected areas such as Cargo and engineering, it also remove the big space that was once the space for the Whiteship, now it's all maintenance

## Why It's Good For The Game

The long awaited sequel to Sp3, it's yet another update for Fland station!

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

![immagine](https://user-images.githubusercontent.com/75247747/152594367-209981cd-8084-4e59-adee-73b7737d3d3b.png)

![immagine](https://user-images.githubusercontent.com/75247747/152594440-54db735d-9086-4d01-a81b-aed1a584099a.png)

![immagine](https://user-images.githubusercontent.com/75247747/152594472-d832f43f-6150-4ddd-bf8e-c8a837096b34.png)

![immagine](https://user-images.githubusercontent.com/75247747/152594530-1aeed356-2e7c-4231-9c4e-58cdaf896aa0.png)

![immagine](https://user-images.githubusercontent.com/75247747/152594618-4c31ae8a-2456-4fa6-8d3c-b1048efacc78.png)

![immagine](https://user-images.githubusercontent.com/75247747/152594689-2754544b-1bb2-4239-bec3-e222d97a28e4.png)

</details>

## Changelog
:cl:
tweak: Converted the auxiliary dock for the whiteship into maint and moved the construction area there, now the whiteship will be able to dock on a dedicated arm westward of the southern arrival arm on Flandstation.
tweak: The teleporter has been moved to where there was the construction area, now it's in a more logical area, in it's place, the library got more maintenance on Flandstation.
tweak: Modified Cargo and Engineering on Flandstation, now they should be slightly better.
tweak: Updated the tech storage, now it should have all the boards needed for a station in case of disaster.
fix: Fixed some odd stuff around Flandstation.
/:cl:

PestoVerde322 committed:
Flandstation SP4 pt1

fuck fuck fuck i put the plush thing togheder

PestoVerde322 committed:
Flandstation Sp4

Changes Cargo, Changes Engineering, see the pull

imsynthex commented on code/game/objects/items/toys.dm:
```diff
@@ -1673,3 +1673,9 @@
+
+/obj/item/rucy
+	name = "PCV Pipe"
+	desc = "Those PCV pipe were the staple of many stations for fluid transmission. You think you hear foreign rap..."
```
```suggestion
	desc = "Those PCV pipes were the staple of many stations for fluid transmission. You think you hear foreign rap..."
```
Also I'm not sure how I feel about these references, but that's not mine to decide 🤷‍♂️

PestoVerde322 committed:
Update code/game/objects/items/toys.dm

Fixed a description

Co-authored-by: ImSynthex <81387903+ImSynthex@users.noreply.github.com>

PestoVerde322 committed:
Fixed some broken turf

PestoVerde322 committed:
Merge remote-tracking branch 'origin/FlandStationSp4' into FlandStationSp4

Archanial commented on code/game/objects/items/toys.dm:
```diff
@@ -1673,3 +1673,9 @@
 	name = "paper - Yatzy Table"
 	info = "<table><tr><th>Upper</th><th>Game 1</th><th>Game 2</th><th>Game 3</th></tr><tr><th>Aces</th><th>\[___\]</th><th>\[___\]</th><th>\[___\]</th></tr><th>Twos</th><th>\[___\]</th><th>\[___\]</th><th>\[___\]</th></tr><th>Threes</th><th>\[___\]</th><th>\[___\]</th><th>\[___\]</th></tr><th>Fours</th><th>\[___\]</th><th>\[___\]</th><th>\[___\]</th></tr><th>Fives</th><th>\[___\]</th><th>\[___\]</t ...
+
+/obj/item/rucy
```
```suggestion
/obj/item/rury
```

PestoVerde322 committed:
Fixups and rury PVC pipe update

PestoVerde322 committed:
Removed the Rury PVC pipe item

Gonna be atomized on it's own PR on Archanial's request

PestoVerde322 committed:
Very minimal change to the b-ball arena in permaroid

PestoVerde322 committed:
Merge branch 'master' of https://github.com/BeeStation/BeeStation-Hornet into FlandStationSp4

PestoVerde322 committed:
Added a flushplush

Archanial commented:
Remove windoors from armory and we're good

unit0016 commented:
Please address the traitor toolbox that just is hanging out in the bridge. Anyone in the know can pop 20TC into this and enjoy having a brute weapon more robust than any other.
![image](https://user-images.githubusercontent.com/50649185/151224621-f9d22fb5-7e9f-4d7d-8498-5329d6dc6723.png)

PestoVerde322 committed:
removed an old toolbox

PestoVerde322 committed:
Updated the tech storage

PestoVerde322 commented:
> Please address the traitor toolbox that just is hanging out in the bridge. Anyone in the know can pop 20TC into this and enjoy having a brute weapon more robust than any other. ![image](https://user-images.githubusercontent.com/50649185/151224621-f9d22fb5-7e9f-4d7d-8498-5329d6dc6723.png)

Just fixed and replaced with a more boring toolbox!

PestoVerde322 committed:
Fixed plumbing in the renewed tech storage

also removed some leftovers, whoops

PestoVerde322 committed:
Fixed some bad piping in the brig area

PestoVerde322 committed:
Other misc stuff being fixed

PestoVerde322 committed:
Added a sink in xenobio

PestoVerde322 committed:
Smol decal fix
